docs/account-validation.md

```diff
@@ -1,101 +1,92 @@
 # Account Validation
 
-Moov Customers project supports following strategies and vendors to validate accounts:
+This project supports the following strategies and vendors to validate accounts:
 
-* micro-deposits validation with Moov [PayGate](./account-validation.md#micro-deposits-validation-with-moov-paygate)
-* instant validation with [Plaid](./account-validation.md#instant-validation-with-plaid)
-* instant validation with [MX](./account-validation.md#instant-validation-with-mx)
+* micro-deposits validation with Moov [PayGate](#micro-deposits-validation-with-moov-paygate)
+* instant validation with [Plaid](#instant-validation-with-plaid)
+* instant validation with [MX](#instant-validation-with-mx)
 
-Please, check [API reference](https://moov-io.github.io/customers/api/#post-/customers/{customerID}/accounts/{accountID}/validate) for more details on account validation API.
+For details on the account validation API, check the [API reference](https://moov-io.github.io/customers/api/#post-/customers/{customerID}/accounts/{accountID}/validations).
 
-[What is Instant Account Validation: MX.com](https://www.mx.com/moneysummit/what-is-instant-account-verification)
+For more information on how instant account validation works, read [What is Instant Account Validation](https://www.mx.com/moneysummit/what-is-instant-account-verification)
 
 ## Micro-deposits Validation with Moov PayGate
 
-In order to validate Account two micro-depoists under $0.50 will be created with a balanced withdraw. Account validation with micro-deposits strategy consists of two steps:
+In order to validate an account, two micro-deposits under $0.50 will be transferred to the customer's bank account.
 
-1. Initiate Account Validation
-2. Complete Account Validation
-
-To use this strategy PayGate should be [configured](../README.md#paygate#paygate).
-
-### Configuration
-
-To use this strategy you have to set `PAYGATE_ENDPOINT` and `PAYGATE_DEBUG_CALLS` environment variable.
+To use this strategy, [configure PayGate](./configuration.md#paygate) and follow the steps below:
 
 ### 1. Initiate Account Validation
 
-During this step Moov Customers creates two micro-deposits for the account via Moov [Paygate](https://github.com/moov-io/paygate).
-
-Here is an example of API call to initiate account validation:
+In this step, Moov Customers creates two micro-deposits for the account through Moov [Paygate](https://github.com/moov-io/paygate).
 
+Here is an example of an API call to initiate account validation:
 ```shell
-curl "http://localhost:8087/customers/51dd8cdd/accounts/b74d7c51/validate" \
--H "Expect:" \
--H 'Content-Type: application/json; charset=utf-8' \
--d @- <<'EOF'
-{
-  "strategy":"micro-deposits",
-  "vendor":"moov"
-}
-EOF
+curl -X POST "http://localhost:8087/customers/51dd8cdd/accounts/b74d7c51/validations" \
+ -H "Accept: application/json, application/json" \
+ -H "X-Request-ID: rs4f9915" \
```

Comment:
Isn't `X-Request-ID` more of a service to service header? Not sure if there's any need to include it in example requests.

Reply:
Good point, I'll remove this.